frameworks/helloworld/cli/dcos-hello-world/main.go

```diff
@@ -2,17 +2,18 @@ package main
 
 import (
 	"github.com/mesosphere/dcos-commons/cli"
+    "github.com/mesosphere/dcos-commons/cli/commands"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 func main() {
 	app := cli.New()
 
-	cli.HandleConfigSection(app)
-	cli.HandleEndpointsSection(app)
-	cli.HandlePlanSection(app)
-	cli.HandlePodsSection(app)
-	cli.HandleStateSection(app)
+	commands.HandleConfigSection(app)
```

Comment:
Something to think about: These explicit section calls were added mainly to get around the downsides of unvendored dependencies. Now that vendoring is being introduced, we could bring back a single `HandleDefaultSections()` call that functions as a shortcut to all of these other calls.